docs/tutorial-create-hello-world.md

```diff
@@ -1,3 +1,58 @@
 # Creating `hello-world.wasm`
 
-... more coming soon
+First, you'll need the standard rust toolchain.
```

Comment:
Could this include a bit of an intro which talks about how there's a number of ways to create wasm files, but for the purposes of this tutorial we're going to be using a Rust toolchain? This can also link to the other section of documentation which details how to build wasm from other languages.